wct.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0"?>
```

Comment:
I guess this should be in the `.gitignore`

Reply:
Thanks for spotting – I've removed and added to `.gitignore`